tests/testthat/test-financialyear.R

```diff
@@ -0,0 +1,24 @@
+library(testthat)
+library(phimethods)
+
```

Comment:
Building the package will load these for you, so the `library` calls can be removed.